x-pack/plugins/security_solution/common/api/timeline/copy_timeline/copy_timeline_route.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
```

Comment:
we now have `zod` in our codebase if you feel like trying something new:)

Reply:
Yeah, hopefully next time :D